### DISCUSSION TITLE
RSS feed of bookmarks/Reading List

### DISCUSSION THREAD

**Discussion by robertandrews** [forem/forem#18376] [Feature Requests]
It would be useful to have this.

**Comment by amywtlin** [comment]
Hi @robertandrews - can you elaborate how you would use this feature? Did you wish to port your reading list to somewhere else?

**Reply by robertandrews** [reply]
Inoreader is an excellent RSS reader. It supports not only RSS feeds but also email newsletters, YouTube etc. I'm trying to use it to accommodate all my content consumption.
Not "port", but just to consolidate the input surface.

**Reply by amywtlin** [reply]
I see. So the desired result is that on Inoreader, you can read all posts you have curated from different sources, the reading list on DEV being one of them. Is that correct?

**Reply by robertandrews** [reply]
> I see. So the desired result is that on Inoreader, you can read all posts you have curated from different sources, the reading list on DEV being one of them. Is that correct?

New posts matching followed topics.

**Comment by SDASDFASDWWWWWWWWWWWWWWWW** [comment]
I totally agreed with you my problem is solved completely. [tarde]([https://doradotarde.com/](https://doradotarde.com/?fbclid=IwAR0CbczFxqKhXfqdRa-J1eMfCVlF_gTT7_nzKjRCl2J6GZ4-PqC1Vj3RZiA)) Thankyou so much. Have a nice day to all…

**Comment by SDASDFASDWWWWWWWWWWWWWWWW** [comment]
I totally agreed with you my problem is solved completely. [tarde ](www.doradotarde.com)Thankyou so much. Have a nice day to all…
